src/Tribe/Tickets_Handler.php

```diff
@@ -1205,7 +1211,12 @@ public function render_ticket_row( $ticket ) {
 			&& -1 !== $capacity
 		) {
 			$product = wc_get_product( $ticket->ID );
-			$needs_warning = (int) $inventory !== (int) $stock;
+			$shared_stock = new Tribe__Tickets__Global_Stock( $ticket->get_event()->ID );
```

Comment:
It's _possible_ `$ticket->get_event()` will return null, so a nice bit of defensive coding could be to protect against that outcome (and remove the potential for an accessing-property-on-non-object error)

Reply:
Yes, it could happen. But only if you edit the DB directly

Reply:
Or if a third party extension / user customization runs amok ;-)